skbio/tree/_tree.py

```diff
@@ -617,6 +617,72 @@ def lowest_common_ancestor(self, tipnames):
 
     lca = lowest_common_ancestor  # for convenience
 
+    def path(self, other, include_ends=False):
+        r"""Return the list of node names in the path from one node to another.
+
+        Parameters
+        ----------
+        other : TreeNode
+            Final node of path.
+        include_ends: bool, optional
+            Whether to include the initial and final nodes in the list.
+
+        Returns
+        -------
+        list
+            List of names of TreeNode objects.
```

Comment:
I think that it should return a list of TreeNodes instead of their names. This is mainly because internal nodes usually don't have unique names, or name at all.